### DISCUSSION TITLE
our-computers-runs-on-sand

### DISCUSSION THREAD

**Discussion by giscus** [scidroid/portfolio#3] [General]
# our-computers-runs-on-sand

Yes, although it seems incredible what I am saying our economy works with sand. well, not exactly, it is a component of sand called silicon.

http://localhost:3000/our-computers-runs-on-sand

**Comment by scidroid** [comment]
Very educative material ✔👏😎🙌👍😊

**Reply by scidroid** [reply]
Yes
